### PR TITLE
JSFiddle: add missing <style> and <script>

### PR DESCRIPTION
**Pull request by Leopoldthecoder**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's [Contributing Guide](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.md).
* [ ] Rebase before creating a PR to keep commit history clear.
* [ ] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by Leopoldthecoder** [comment]
好的，现在例子都能在 JSFiddle 里正常运行了。

**Comment by QingWei-Li** [comment]
6666

File: examples/docs/en-US/menu.md [hunk 1]
```diff
@@ -45,8 +45,17 @@ Top bar NavMenu can be used in a variety of scenarios.
   </el-submenu>
   <el-menu-item index="3">Orders </el-menu-item>
 </el-menu>
-```
 
+<script>
+  export default {
+    methods: {
+      handleselect(key, keyPath) {
```
Comment (csvwolf) [inline examples/docs/en-US/menu.md]: 为啥这个页面的`handle`画风和别人不一样

File: examples/docs/zh-CN/menu.md [hunk 2]
```diff
@@ -75,6 +75,16 @@
   </el-submenu>
   <el-menu-item index="3">订单管理</el-menu-item>
 </el-menu>
+
+<script>
+  export default {
+    methods: {
+      handleselect(key, keyPath) {
```
Comment (csvwolf) [inline examples/docs/zh-CN/menu.md]: +1

Comment (csvwolf) [inline examples/docs/zh-CN/menu.md]: =_=手滑点成approve了

File: examples/docs/en-US/menu.md (hunk 1 above)
Comment (Leopoldthecoder) [inline examples/docs/en-US/menu.md]: 更新了。

**Pull request merged**